packages/wdio-sync/src/index.js

```diff
@@ -75,3 +75,9 @@ export {
     executeSync,
     runSync,
 }
+
+export default function sync (testFn) {
```

Comment:
why do we need another export?

Reply:
given that @wdio/sync is no suppose to be used by end user I thought it would be nice if people can just do import sync from '@wdio/sync' when they want to use it.